src/search/pdbs/cegar.cc

```diff
@@ -532,28 +532,28 @@ void add_implementation_notes_to_parser(options::OptionParser &parser) {
         "Implementation Notes",
         "The following describes differences of the implementation to "
         "the original implementation used and described in the paper.\n\n"
-        "Conceptually, there is one larger difference which concern the "
+        "Conceptually, there is one larger difference which concerns the "
         "computation of (regular or wildcard) plans for PDBs. The original "
-        "implementation used an enforced hill-climbing search with the PDB "
-        "as the perfect heuristic. To handle domains with zero-cost operators, "
-        "it used a breadth-first search in each iteration to find the best "
-        "improving successor. This ensured to compute strongly optimal plans, "
-        "i.e., optimal plans with a minimum number of zero-cost operators. "
-        "The original implementation also ensured to use uniform random tie-"
-        "breaking when choosing the best successor.\n\n"
+        "implementation used an enforced hill-climbing (EHC) search with the "
```

Comment:
the word "PDB" got lost here

Reply:
Actually, I forgot to move it (it was still there, but at the wrong position).